### ISSUE TITLE
app_links: 3.4.0 No podspec found for macOS

### ISSUE THREAD

**Issue by pro100andrey** [llfbandit/app_links#37]
```yaml
//pubspec.yaml file

dependencies:
  app_links: ^3.4.0
```

**How to reproduce:**
``` bash
cd  macos
rm Podfile.lock
pod install --repo-update
```

**Result:**

[!] No podspec found for `app_links` in `Flutter/ephemeral/.symlinks/plugins/app_links/macOS`

Thanks

**Comment by llfbandit** [comment]
Sorry for that, it should now be fixed in master.
Can you give it a try before publication?

**Comment by dbrdz** [comment]
I was having this same problem, the problem is not present in master. Thank you!

**Comment by pro100andrey** [comment]
Hi @llfbandit, thanks for quick response!

After connect master:

``` yaml
app_links: 
    git:
      url: "https://github.com/llfbandit/app_links"
      path: app_links
```

 this issue can't reproduce.

Thanks!

**Comment by llfbandit** [comment]
The fix is now available in version 3.4.1.
Many thanks for the report and feedback to all of you!
